openmls/src/treesync/mod.rs

```diff
@@ -68,6 +71,102 @@ pub use node::{leaf_node::LeafNode, parent_node::ParentNode, Node};
 #[cfg(any(feature = "test-utils", test))]
 pub mod tests_and_kats;
 
+/// An exported ratchet tree as used in, e.g., [`GroupInfo`](crate::messages::group_info::GroupInfo).
+#[derive(
+    PartialEq,
+    Eq,
+    Clone,
+    Debug,
+    Default,
+    Serialize,
+    Deserialize,
+    TlsSerialize,
+    TlsDeserialize,
+    TlsSize,
+)]
+// TODO(validation):
```

Comment:
We could already open an issue for that.

Reply:
Yes, please convert this into TODOs with a ticket number

Reply:
I created #1305.